change/@ni-nimble-components-46b1fdfb-63b3-4a04-ac71-1124855a4e45.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fix for Firefox caret disappearing issue",
+  "packageName": "@ni/nimble-components",
+  "email": "aagash.maridas@solitontech.com",
```

Comment:
nit: Would you like to obfuscate your email address based on [this suggestion](https://github.com/ni/nimble/blob/4574201ae49f8e30da7a0d87db78109f4236f340/CONTRIBUTING.md?plain=1#L79)? 

https://github.com/ni/nimble/blob/4574201ae49f8e30da7a0d87db78109f4236f340/CONTRIBUTING.md?plain=1#L79